game/Bird.java

```diff
@@ -101,7 +101,7 @@ public void update() {
         if (dead) return;
 
         double birdYPossition = (double)yPos/Resources.HEIGHT;
-        double nextTubeHeigth = ((Resources.nextTube.getY() + Resources.TUBE_HEIGHT + Resources.TUBE_GAP_DISTANCE/2)-560+Resources.TUBE_HEIGHT)/(-260+560); // 80-380
+        double nextTubeHeigth = ((Resources.nextTube.getY() + Resources.TUBE_HEIGHT + Resources.TUBE_GAP_DISTANCE/2)-560+Resources.TUBE_HEIGHT)/(700); // 80-380
```

Comment:
This will not evaluate to the same value (original's divide is 300, where as the replacement is 700), also the parentheses on the right can be removed.